README.md

```diff
@@ -1 +1,17 @@
 # TLA+ Foundation
+
+# 1. Goals of the Foundation
+The TLA+ Foundation (TLAF) is an independent, non-profit organization dedicated to fostering the adoption of the TLA+ specification language in industry, academia, and education, guided by the overall goal to advance the practice of mathematically thinking in software engineering.  The mission of TLAF is active support and funding of the development of the TLA+ tools to broaden their use by practitioners.  If not directly funding work, TLAF creates synergies whenever possible.
```

Comment:
"mathematically thinking in software engineering" -> "mathematical thinking in software engineering"